### PR TITLE
hiding button if no attires present

### LINKED ISSUE #119: Attires button is always being displayed
If the boards doesn't have an attire, we shouldn't display the button.
@rodri042 Could you provide a method in gs-board to check this?

### PR DESCRIPTION
**Pull request by luchotc**
I don't like this approach at all, so if you have a better one please change it @rodri042 

Resolves #119

### CODE REVIEW

File: lib/render/editor/editor.html [hunk 1]
```diff
@@ -626,10 +628,17 @@
         }
       },
 
+      ready: function () {
+        this.boards = $(".mu-kids-states").find("gs-board");
+        setTimeout(() => {
+          this.shouldDisplay = this.boards.toArray().some( (board) => { return board.attire });
```
[the remark this answers is not in the document]
Comment (luchotc) [reply lib/render/editor/editor.html]: Already tried it but the problem is that the boards are not initialized yet by then

File: lib/render/editor/editor.html [hunk 2]
```diff
@@ -626,10 +628,17 @@
         }
       },
 
+      ready: function () {
+        this.boards = $(".mu-kids-states").find("gs-board");
+        setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (luchotc) [reply lib/render/editor/editor.html]: this.shouldDisplay = this.boards.toArray().some( (board) => { return board.attire });

[the remark this answers is not in the document]
Comment (luchotc) [reply lib/render/editor/editor.html]: Already tried it but the boards aren't defined yet using that approach. 
I don't know much about Polymer, but can't we subscribe to an event in the gs-board somehow?

[the remark this answers is not in the document]
Comment (rodri042) [reply lib/render/editor/editor.html]: I didn't find a way to do that yet, but `async` used to work. You could try listening to `board-changed` adding an event listener, the board always fires the event when it starts

https://www.polymer-project.org/1.0/docs/devguide/events

**Comment by flbulgarelli** [comment]
@luchotc can this PR be merged? Are you going to change to `async`  as suggested by @rodri042 or are we going to postpone this change?

**Comment by luchotc** [comment]
@flbulgarelli I'm on it right now. Async doesn't work because boards aren't initialized yet and i tried listening to board-changed but it works only sometimes, as there's a "race condition" between gs-board and this component.
The current code works flawless but i don't like having an arbitrary timeout of half a second.

If i don't find a better solution, i think it should be merged anyway, it's annoying having a button that does nothing (when the boards don't have attires)

**Pull request merged**